packages/block-library/src/social-link/edit.js

```diff
@@ -139,8 +139,7 @@ const SocialLinkEdit = ( {
 				<PanelBody title={ __( 'Settings' ) }>
 					<PanelRow>
 						<TextControl
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
Insert "Social Icons" block and add a social.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/33f4b25b-95be-4c2b-bfcd-f8dfe057b820" alt="Social Icon block inspector, before" width="274">|<img src="https://github.com/user-attachments/assets/62cc283b-786b-4fe4-8fa9-7f588bc0a373" alt="Social Icon block inspector, after" width="274">|